validation/airgap/provisioning/k3s_custom_cluster_test.go

```diff
@@ -73,39 +75,23 @@ func (a *AirGapK3SCustomClusterTestSuite) SetupSuite() {
 	newUser, err := users.CreateUserWithRole(client, user, "user")
 	require.NoError(a.T(), err)
 
-	standardUserClient, err := client.AsUser(newUser)
-	require.NoError(a.T(), err)
-
-	a.client = standardUserClient
-
-	listOfCorrals, err := corral.ListCorral()
-	require.NoError(a.T(), err)
-
-	corralConfig := corral.Configurations()
-
-	err = corral.SetupCorralConfig(corralConfig.CorralConfigVars, corralConfig.CorralConfigUser, corralConfig.CorralSSHPath)
+	a.standardUserClient, err = client.AsUser(newUser)
 	require.NoError(a.T(), err)
 
-	a.corralPackage = corral.PackagesConfig()
-
-	_, corralExist := listOfCorrals[corralRancherHA.Name]
-	if corralExist {
-		bastionIP, err := corral.GetCorralEnvVar(corralRancherHA.Name, corralRegistryIP)
-		require.NoError(a.T(), err)
-
-		err = corral.UpdateCorralConfig(corralBastionIP, bastionIP)
-		require.NoError(a.T(), err)
-
-		registryFQDN, err := corral.GetCorralEnvVar(corralRancherHA.Name, corralRegistryFQDN)
-		require.NoError(a.T(), err)
-		logrus.Infof("registry fqdn is %s", registryFQDN)
+	corralRancherHA := new(corralha.CorralRancherHA)
+	config.LoadConfig(corralha.CorralRancherHAConfigConfigurationFileKey, corralRancherHA)
+	if corralRancherHA.Name != "" {
+		a.registryFQDN = airgap2.AirgapCorral(a.T(), corralRancherHA)
```

Comment:
are we leaving in the old airgap for some reason? imo it can completely be removed; hostbusters was the only one using it once-upon-a-time, and they haven't used it in some time. so no need for airgap2, we should be able to completely rip the old version. 

<edit> actually I'm confused, why is this called airgap2 since corral is still being used here